src/main/java/athena/Athena.java

```diff
@@ -27,11 +27,14 @@ public static void main(String[] args) {
         command.execute(taskList, athena.ui);
         command = parser.parse("list");
         command.execute(taskList, athena.ui);
-        command = parser.parse("help");
-        command.execute(taskList, athena.ui);
+//        command = parser.parse("help");
+//        command.execute(taskList, athena.ui);
         command = parser.parse("list");
         command.execute(oldTask, athena.ui);
         storage.saveTaskListData(taskList);
+        command= parser.parse("delete 2");
+        command.execute(taskList, athena.ui);
+        storage.saveTaskListData(taskList);
```

Comment:
We should put these things in our JUnit tests. I think it's ok if you want to do it in a different pull request.